docker-compose.yml

```diff
@@ -1,85 +1,108 @@
 
 services:
-  nginx:
-    build: ./docker/nginx
-    ports:
-      - "443:443"
-    restart: "on-failure"
+#   nginx:
+#     build: ./docker/nginx
+#     ports:
+#       - "443:443"
+#     restart: "on-failure"
```

Comment:
As far as I understand from the description, you commented this just for testing and will change it back before merging? If you're looking for someone to check that the setup works, then I can happily give it a try on the weekend

Reply:
Yes, exactly, I will comment this code back in before merging. This is just for testing, so that you can use "docker compose up" to start the test script. Will remove the commit that commented this code and force push the branch after the review is done.